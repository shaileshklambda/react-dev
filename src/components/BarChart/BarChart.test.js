import React from "react";
import renderer from "react-test-renderer";
import Barchart from "./index";

it("Barchart: default", () => {
  const pref = {
    chartData: {},
    chartOptions: {}
  };
  const component = renderer.create(<Barchart pref={pref} />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
