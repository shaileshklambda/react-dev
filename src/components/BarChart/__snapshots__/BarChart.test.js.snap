// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Barchart: default 1`] = `<div />`;
